example/android/app/google-services.json

```diff
@@ -1,68 +0,0 @@
-{
-  "project_info": {
-    "project_number": "123",
-    "project_id": "spike-iterable",
-    "storage_bucket": "spike.appspot.com"
```

Comment:
Why do you want to delete this file?

Reply:
Sorry, thought I had accidentally pushed up my google-services.json file. I've since reverted this change.